build.gradle

```diff
@@ -0,0 +1,123 @@
+plugins {
+	id 'org.standardout.bnd-platform'
+}
+
+repositories {
+	mavenCentral()
+}
+
+defaultTasks 'clean', 'updateSite'
+
+platform {
+	categoryId = platformCategoryId
+	categoryName = platformCategoryName
+
+	useBndHashQualifiers = false
+	useFeatureHashQualifiers = false
+	generatePlatformFeature = false
+	eclipseMirror = eclipseDownloadUrl
+
+	feature(id: 'org.javatuples.feature',
+			name: 'Javatuples Feature',
+			version: javatuplesVersion,
+			provider: providerName,
+			license: 'https://raw.githubusercontent.com/javatuples/javatuples/master/LICENSE.txt'.toURL().getText()) {
```

Comment:
I like the idea of not statically writing down the license. Anyway, referring to master could be problematic if the license changes in future or the repository is deleted (same for others).

Linking to a tag is possible but this requires a proper naming scheme in the third-party source code management that allows integrating the version that has to be set in the properties file. We should check if this is possible.

Unfortunately, I have no real solution because you would have to read the artifact meta data from maven central, parse the referenced URL and use it here. Looks like a big overhead for a simple fature...

Reply:
You are right. We should not reference to the master. I will try to find version appropriate tags to reference to. Then a version change should be not a problem. If the repo is removed, the license cannot be resolved and the build will fail. IMHO, this should be sufficient.

Reply:
For each depencency I am now referencing a version tagged branch on github. At least a license change should now be mitigated.